### PR TITLE
ivanti-connect-secure-xsiam-modeling

### PR DESCRIPTION
**Pull request by cweltPA**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-6571

## Description
A new XSIAM pack for ICS (Ivanti Connect Secure). 

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/IvantiConnectSecure/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+
+# Ivanti Connect Secure
+<~XSIAM>
+This pack includes Cortex XSIAM content.
+
+
+## Configuration on Server Side
+This section describes the mandatory steps you should perform on Ivanti Connect Secure admin console in order to configure logging for the various system events and forwarding them via Syslog to XSIAM.
```
Comment (ShirleyDenkberg) [inline Packs/IvantiConnectSecure/README.md:8]:
```suggestion
This section describes the mandatory steps you should perform on Ivanti Connect Secure admin console in order to configure logging for the various system events and forwarding them via Syslog to Cortex XSIAM.
```

File: Packs/IvantiConnectSecure/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+### Configure Events to Log and Syslog forwarding 
+1. Log in to your Ivanti Connect Secure admin web console. 
+2. Select **System** &rarr; **Log/Monitoring**.
```
Comment (ShirleyDenkberg) [inline Packs/IvantiConnectSecure/README.md:12]:
```suggestion
2. Select **System** > **Log/Monitoring**.
```

File: Packs/IvantiConnectSecure/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+3. Click the **Settings** tab to display the configuration page.
+4. For each local event log category that you are interested in logging (_System_ events, _User Access_ events, _Admin Access_ events & _Sensors_ events), perform the following steps: 
+     - Go to the **Select Events to Log** section, and select the requested event types you wish to log. See [Select Events to Log](https://help.ivanti.com/ps/help/en_US/PCS/9.1R13/pcsag/logging_n_monitoring.htm#log_events_set) for a description of the various types for each event category. 
+     - Go to the  **Syslog Server** section, and define a server configuration for the XSIAM Broker VM with the following settings (see [Configuring Syslog](https://help.ivanti.com/ps/help/en_US/PCS/9.1R13/pcsag/logging_n_monitoring.htm#logging_and_monitoring_1494202195_1023612) for full documentation):
```
Comment (ShirleyDenkberg) [inline Packs/IvantiConnectSecure/README.md:16]:
```suggestion
     - Go to the  **Syslog Server** section, and define a server configuration for the Cortex XSIAM Broker VM with the following settings (see [Configuring Syslog](https://help.ivanti.com/ps/help/en_US/PCS/9.1R13/pcsag/logging_n_monitoring.htm#logging_and_monitoring_1494202195_1023612) for full documentation): 
```

File: Packs/IvantiConnectSecure/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+1. Log in to your Ivanti Connect Secure admin web console. 
+2. Select **System** &rarr; **Log/Monitoring**.
+3. Click the **Settings** tab to display the configuration page.
+4. For each local event log category that you are interested in logging (_System_ events, _User Access_ events, _Admin Access_ events & _Sensors_ events), perform the following steps: 
+     - Go to the **Select Events to Log** section, and select the requested event types you wish to log. See [Select Events to Log](https://help.ivanti.com/ps/help/en_US/PCS/9.1R13/pcsag/logging_n_monitoring.htm#log_events_set) for a description of the various types for each event category. 
+     - Go to the  **Syslog Server** section, and define a server configuration for the XSIAM Broker VM with the following settings (see [Configuring Syslog](https://help.ivanti.com/ps/help/en_US/PCS/9.1R13/pcsag/logging_n_monitoring.htm#logging_and_monitoring_1494202195_1023612) for full documentation): 
+         | Parameter          | Value    
+         | :---               | :---                    
+         | `Server name/IP`   | Enter the [XSIAM Broker VM](#broker-vm) server's IP address or FQDN.
```
Comment (ShirleyDenkberg) [inline Packs/IvantiConnectSecure/README.md:19]:
```suggestion
         | `Server name/IP`   | Enter the [Cortex XSIAM Broker VM](#broker-vm) server's IP address or FQDN.    
```

File: Packs/IvantiConnectSecure/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+## Collect Events from Vendor
+In order to use the collector, use the [Broker VM](#broker-vm) option.
+
+### Broker VM
+To create or configure the Broker VM, use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Configure-the-Broker-VM).
+
+You can configure the specific vendor and product for this instance.
+
+1. Navigate to **Settings** &rarr; **Configuration** &rarr; **Data Broker** &rarr; **Broker VMs**.
```
Comment (ShirleyDenkberg) [inline Packs/IvantiConnectSecure/README.md:38]:
```suggestion
1. Navigate to **Settings** > **Configuration** > **Data Broker** > **Broker VMs**. 
```

**Comment by ShirleyDenkberg** [comment]
@cweltPA Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**